src/pages/help/faq/index.mdx

```diff
@@ -121,12 +121,31 @@ The Carbon Native mobile Figma kit is available on the
 [Other resources](https://carbondesignsystem.com/designing/design-resources/#native-mobile)
 page.
 
-### Where do I find the Experimental components and patterns?
-
-The patterns and components that were in Experimental are now in the
-[Community assets](/community/overview) section. Community assets include work
-developed and maintained by the individuals and teams who use Carbon. You can
-help elevate these resources by contributing feedback, code, and design.
+### What are feature flags? Where do I find them?
+
+Feature flags enable new behaviors and styling, allowing users to opt-in to new
+breaking changes while staying on the current major version. They are currently
+available on the Component Overview page
+under [Feature flags](https://carbondesignsystem.com/components/overview/feature-flags/)
+tab.
+
+### What are experimental components? Where do I find them?
+
+The team occasionally will author code, or accept contributions, that is
+considered experimental or unstable. The goal for this code is to ship it as
+unstable for sponsor groups to leverage. During this time, the team can get
+feedback around what is working and what does not work so that changes can be
+made before an official release. They are available on the storybook under the
+[Experimental](https://react.carbondesignsystem.com/?path=/docs/experimental-feature-flags--overview)
```

Comment:
As Alison said, we will need a new link to put here.

Reply:
Alison mentioned that it's a bug, so I'm assuming the link will stay the same once it's fixed. I'm still waiting for the dev team to address it, so I've added a "DO NOT MERGE" tag for now. I'll check back on it after the fix.